js/common/lib/tensor.ts

```diff
@@ -307,8 +307,8 @@ export interface TensorFromImageOptions {
    * Describes normalization parameters to tensor conversion from image data - default values - Bias: 0, Mean: 255
    */
   norm?: {
-    bias?: number;  // Todo add support - |[number,number,number]|[number,number,number,number];
-    mean?: number;  // Todo add support - |[number,number,number]|[number,number,number,number];
+    bias?: number|[number, number, number]|[number, number, number, number];
```

Comment:
add jsdoc to explain that bias and mean can be either a number or an array of numbers with length of 3 or 4. 

Those comments added into jsdoc will reflect in the generated API docs so that is why need it to be more detailed

Reply:
Right - I will updated the documentation to include the right description + an example call